src/cocotb_tools/runner.py

```diff
@@ -561,10 +588,16 @@ def _test_command(self) -> List[Command]:
         ]
 
     def _build_command(self) -> List[Command]:
-        if self.vhdl_sources:
-            raise ValueError(
-                f"{type(self).__qualname__}: Simulator does not support VHDL"
-            )
+        for source in self.sources:
+            if not is_verilog_source(source):
```

Comment:
We may want to support more file types at some point, so ideally the decision logic here would already be prepared for when that time comes.

Reply:
That would be preferable, but I think here since we only support Verilog sources on this simulator, this is appropriate. Other places, like filtering the build args `not isinstance(arg, VHDL)` is less precise than `isinstance(arg, Verilog) or type(arg) is str`.